scripts/ddp-RxTerms-script/src/index.ts

```diff
@@ -0,0 +1,53 @@
+import * as XLSX from "xlsx";
+import { RxNormDrugOption } from "./models";
+import { parseToHOCON, writeFile } from "./utils/common";
+
+const inputFileName = process.argv[2] || "RxTerms202202.txt";
+const outputFileName = process.argv[3] || "output.txt";
+
+const translations: string[] =  process.argv.filter(arg => ['en', 'es', 'ru'].includes(arg));
```

Comment:
I think we are unnecessarily limiting the the codes that are acceptable.
Just do something like this:
```typescript
const languageCodes: string[] | void = process.argv[4]?.split(",");

if (!languageCodes) {
    console.error("Comma separated language separators were not provided");
    process.exit(1)
}

Now users can say en,es,fn,etc and we will use it.